api/models/devices.go

```diff
@@ -0,0 +1,195 @@
+package models
+
+import (
+	"fmt"
+	"bytes"
+	"errors"
+  "github.com/spf13/viper"
+	"github.com/TheThingsNetwork/ttn/core"
+	"github.com/TheThingsNetwork/ttn/core/components/handler"
+	"github.com/TheThingsNetwork/ttn/utils/random"
+	"github.com/TheThingsNetwork/ttn/api/auth"
+	"golang.org/x/net/context"
+	"encoding/json"
+)
+
+type Device struct {
+	DevEUI   EUI     `json:"eui,omitempty"`
+	AppEUI   EUI     `json:"app"`
+	Type     string  `json:"type"`
+	Address  []byte  `json:"address"`
+	FCntUp   uint32  `json:"fcnt_up"`
+	FCntDown uint32  `json:"fcnt_down"`
+	Active   bool    `json:"activated"`
+	AppSKey  SKey    `json:"app_secret"`
+	NwkSKey  SKey    `json:"nwk_secret"`
```

Comment:
Introduce type alias for both types of keys

Use JSON field names that match field names of structure